### PR TITLE
feat-typography

### PR DESCRIPTION
**Pull request by abhij1607**
image

### CODE REVIEW

File: components/typography/typography.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
+h1,
+.h1 {
+  font-size: 3.2rem;
+  margin: 10px 0;
+}
+h2,
+.h1 {
```
Comment (Hamdrive) [inline components/typography/typography.css]: I believe this is supposed to be h2 class, typo

File: components/typography/typography.css [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+.p-xsm {
+  font-size: 5px;
+}
+.text-wt-lt {
+  font-weight: 300;
+}
+.text-wt-md {
+  font-weight: 500;
+}
+.text-wt-bold {
```
Comment (Hamdrive) [inline components/typography/typography.css]: Refer to the Figma file, you can use the same font weight class names there
`txt-light, txt-md, txt-bold` etc

**Review by Hamdrive — CHANGES_REQUESTED** [review]
Good work, have suggested alternative for class names

Also suggested to fix the typo in classname as mentioned

**Review by junaidshaikh-js — APPROVED** [review]
Looks good to me! All the best practices followed, can be merged.

File: components/typography/typography.css (hunk 1 above)
Comment (abhij1607) [reply components/typography/typography.css]: thanks for pointing it out, resolved

File: components/typography/typography.css (hunk 2 above)
Comment (abhij1607) [reply components/typography/typography.css]: Thanks for suggestion

**Pull request merged**